src/components/combobox/combobox.e2e.ts

```diff
@@ -514,7 +516,7 @@ describe("calcite-combobox", () => {
       await page.waitForChanges();
       expect(await page.evaluate(() => document.activeElement.id)).toBe("myCombobox");
 
-      await page.keyboard.press("ArrowDown");
+      await page.keyboard.press("Space");
```

Comment:
Do we also need a test for `ArrowDown` opening the menu?

Reply:
💡 There is a test below this one that does that.